### DISCUSSION TITLE
Is there a way to use and set up tailwindcss v3 without using cra 5 alpha?

### DISCUSSION THREAD

**Discussion by nasratt** [tailwindlabs/tailwindcss#6509] [Help]
I want to use the tailwindcss v3 in my react app, but I don't want to use the car 5 alpha, is there a way to set it up that way?

**Comment by lukewarlow** [comment]
Adam has setup a [demo project](https://github.com/adamwathan/cra-tailwind-3) that uses CRA 4 and the Tailwind CLI. That should be exactly what you're looking for.
